packages/lib/test/contracts/ProxyAdmin.test.js

```diff
@@ -40,7 +38,7 @@ contract('ProxyAdmin', function(accounts) {
       this.ownable = this.proxyAdmin;
     });
 
-    shouldBehaveLikeOwnable(proxyAdminOwner, anotherAccount);
+    // shouldBehaveLikeOwnable(proxyAdminOwner, anotherAccount);
```

Comment:
👀  Why is that?

Reply:
Ha! nice catch. I left it commented while debugging :cry: